data/campaigns/Under_the_Burning_Suns/units/quenoth/Sun_Singer.cfg

```diff
@@ -19,7 +19,7 @@
     [abilities]
         {UTBS_ABILITY_HEALS}
     [/abilities]
-    description= _ "Faerie and elven magic have oft been aptly associated with life, from which they draw the majority of their power. However, with the death of much of Irdya’s flora, the Quenoth Elves were forced to seek another source for their sorcery. In time, they learned to harness the power of the suns Sela and Naia - the twin stars that had razed the forests of Irdya to ashes, yet still spring forth the energy required for sparking life. Those who master this new magic sing of the dual nature of these embodiments of fire: flames that are both life and life’s demise."
+    description= _ "Faerie and elven magic have oft been aptly associated with life, from which they draw the majority of their power. However, with the death of much of Irdya’s flora, the Quenoth Elves were forced to seek another source for their sorcery. In time, they learned to harness the power of the suns Sela and Naia — the twin stars that had razed the forests of Irdya to ashes, yet still spring forth the energy required for sparking life. Those who master this new magic sing of the dual nature of these embodiments of fire: flames that are both life and life’s demise."
```

Comment:
"both life and life's demise" seems unbalanced. Suggestion: "both life's kindling and life's demise".

Reply:
Out of scope for this PR.

Reply:
While it is out of this PR's scope, it's something where any changes should be done before the string is frozen. I'd suggest either merging this PR quickly so that we can open campaign-specific PRs, or closing this PR and replacing it with PRs that can have such changes in-scope. Preferably the quick-merge, so that the already reviewed changes don't reappear in another PR.

Reply:
I'm happy to have #7964 and this merged so that any further changes as what you suggest can be done.